### PR TITLE
The Krisitik Species

### PR DESCRIPTION
**Pull request by CharlesWedge**
## For your consideration a Rodent Race
The Krisitik are a race of subterrianian murine aliens from the Saggitarius Arm. Having been forced underground by the toxic and radioactive surface of their homeworld they have spent a millenia perfecting nuclear technology to power their massive underground cities. With their government slowly turning away form their previous isolationism and protectionism, the Krisitik are being seen more and more in the wider galaxy including on Nanotrasen stations.

## Why It's Good For The Game
Krisitik are something I have been working on for a while and I tend to use them as a springboard to develop further neat things further on. These may include more plants for botany, additional subspecies, new Cooking Recipes, special cargo only items, and other special equipment.

## Changelog

:cl:
add: The Krisitik Species, may need a slight amount of balancing going forward.
tweak: Squeakish Language Rework, hopefully it sounds slightly less ridiculous now.
/:cl:

### CODE REVIEW

File: code/modules/catalogue/catalogue_data.dm [hunk 1]
```diff
@@ -229,6 +229,24 @@ GLOBAL_DATUM_INIT(catalogue_data, /datum/category_collection/catalogue, new)
...
+	reliance on nuclear energy. \
+	<br><br>\
+	Their society is highly competitive and the frequent disasters in their underground \
+	cities mean the species as a whole has a degree of paranoia far above average for \
+	most sapients, instinctively prepping for any kinds of disaster. \
+	<br><br>\
+	Even before their discovery of FTL the Krisitik had a very advanced society \
+	the discovery of FTL was likely delayed due to the subterrian nature of Krisitik \
+	society and the toxic atmosphere of their homeworld. This edge has meant the Krisitik \
+	have faired better as an independent species then other 'newcomers' to the galaxy. "
```
Comment (Shadowtail117) [inline code/modules/catalogue/catalogue_data.dm:247]:
```suggestion
	desc = "Krisitik are a species of subterranean mammals from the planet of Murith. \
	Their unstable genetics mean the species has a varied appearance, though they \
	can generally be described as murine. Their unstable genetics also mean \
	they are prone to cancers and lethal mutations, which is aggravated by the species' \
	reliance on nuclear energy. \
	<br><br>\
	Their society is highly competitive, and the frequent disasters in their underground \
	cities mean the species as a whole has a degree of paranoia far above average for \
	most sapients, instinctively prepping for any kind of disaster. \
	<br><br>\
	Even before their discovery of FTL, the Krisitik had a very advanced society. \
	Their discovery of FTL was likely delayed due to the subterranean nature of Krisitik \
	society and the toxic atmosphere of their homeworld. This edge has meant the Krisitik \
	have faired better as an independent species than other 'newcomers' to the galaxy. "
```

File: code/modules/species/station/standard/krisitik.dm [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	blurb = {"
+	Krisitik are a race of subterrainian dwelling murine people from the Perseus Arm. Already significantly advanved
+	even before developing FTL, the Krisitik have exploded onto the galaxy using their mastery of nuclear technology
+	to do their best to bridge the gap between them and the galactic superpowers. The crowding in the massive underground\
+	cities of their toxic homeworld of Murith has lead many Krisitik to emigrate to their colonies or towards the
+	corporate frontier. As their isolationist government becomes more open, this emigration is only increasing.
+
+	The Krisitik are highly varied people owing to their unstable genetics which can result in lethal defects
+	if left untreated. They are a competitive, survivalist and paranoid people known to stretch bend and
+	break the rules if it means being prepared for disasters to come, even if they are unlikely."}
```
Comment (Shadowtail117) [inline code/modules/species/station/standard/krisitik.dm:33]:
```suggestion
	Krisitik are a race of subterranean murine people from the Perseus Arm. Already significantly advanced
	even before developing FTL, the Krisitik have exploded onto the galaxy using their mastery of nuclear technology
	to bridge the gap between them and the galactic superpowers. The crowding in the massive underground\
	cities of their toxic homeworld of Murith has led many Krisitik to emigrate to their colonies or towards the
	corporate frontier. As their isolationist government becomes more open, this emigration is only increasing.

	The Krisitik are highly varied people owing to their unstable genetics, which can result in lethal defects
	if left untreated. They are a competitive, survivalist, and paranoid people known to stretch, bend, and
	break the rules if it means being prepared for disasters to come, even if they are unlikely."}

```

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
grammar/spelling fixes

File: code/modules/species/station/standard/krisitik.dm [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	base_color  = "B2B2B2"
+
+	slowdown  = -0.5
+
+	brute_mod = 1.1
+	burn_mod  = 1.1
+	flash_mod = 1.5 //Cave creatures don't like bright lights
+	radiation_mod = 0.75 //Hardened Genes
+
+	gluttonous    = 1
```
Comment (TheLordME) [inline code/modules/species/station/standard/krisitik.dm:53]: I am not sure if this is the var you are intending to use here:
`/// Can eat some mobs. 1 for mice, 2 for monkeys, 3 for people.
var/gluttonous`
if you want to make the hungry faster or slower
use hunger_factor (0.05 is baseline, 0.2 is tesh, 0.04 is slow metabolism)

File: code/modules/species/station/standard/krisitik.dm [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+	catalogue_data = list(/datum/category_item/catalogue/fauna/krisitik)
+
+
+	species_spawn_flags = SPECIES_SPAWN_CHARACTER
+	species_appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	flesh_color = "#AFA59E"
+	base_color  = "B2B2B2"
+
+	slowdown  = -0.5
```
[the remark this answers is not in the document]
Comment (TheLordME) [reply code/modules/species/station/standard/krisitik.dm:46]: xenohybrids use -0.2, naramadi -0.25
just for consideration

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/modules/species/station/standard/krisitik.dm (hunk 3 above)
Comment (CharlesWedge) [reply code/modules/species/station/standard/krisitik.dm:53]: Krisitik would probably have no issue eating mice to be honest, even if its not considered 'polite'. I am keeping this.

File: code/modules/species/station/standard/krisitik.dm (hunk 4 above)
[the remark this answers is not in the document]
Comment (CharlesWedge) [reply code/modules/species/station/standard/krisitik.dm]: I have lowered it to -.35 they are meant to be speedy but yes they can afford to be slowed down a bit

File: code/modules/species/station/standard/krisitik.dm (hunk 3 above)
Comment (TheLordME) [reply code/modules/species/station/standard/krisitik.dm:53]: I just wanted to point out that there is a different variable you might have been aiming at

**Review by TheLordME — APPROVED** [review]

**Pull request merged**